Docs/Navigation.md

```diff
@@ -172,43 +172,92 @@ A simple view that will render a scene for the currently presented sub-state. Th
 
 [NavigationAnimatedView](AnimatedView.md) is the spiritual successor to Navigator. In addition to adopting a declaritive API, it uses the Animated library to delegate animations and gestures to the scenes. 
 
-NavigationCard and NavigationHeader are the included implementations of scenes and overlays for NavigationAnimatedView, which are intended to look similar to platform conventions.
+Basically, NavigationAnimatedView acts as a controlled component that takes the `navigationState` and redners the scenes and header (as overlay).
+
+```
+<NavigationAnimatedView
+  navigationState={navigationState}
+  renderScene={renderScene}
+  renderOverlay={renderOverlay}
+/>
+```
+
+The interface of the functions `renderScene` and `renderOverlay` should be defines as the following snippets:
+
+```
+function renderOverlay(props: NavigationSceneRendererProps): ReactElement {
+  return <View><Text>overlay {props.scene.navigationState.key}</Text></View>
+}
+
+function renderScene(props: NavigationSceneRendererProps): ReactElement {
+  return <View><Text>scene {props.scene.navigationState.key}</Text></View>
+}
+```
+
+Note that the param `props` is prepared by the NavigationAnimatedView as read-only object that contains the information we need to render the scenes, headers.
```

Comment:
`props` is a read-only object that contains the information needed to render the scenes and headers.